dll/ntdll/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ add_subdirectory(nt_0600)
 
 spec2def(ntdll.dll def/ntdll.spec ADD_IMPORTLIB)
 
+# Use a CRT-only import lib for modules needing it
+generate_import_lib(libntdll_crt ntdll.dll def/ntdll_crt.spec)
+
+# We link ntdll_crt to modules not linking to msvcrt
+target_link_libraries(libntdll INTERFACE "$<$<NOT:$<IN_LIST:libmsvcrt,$<TARGET_PROPERTY:LINK_LIBRARIES>>>:libntdll_crt>")
```

Comment:
With this, people could still forget to link msvcrt and would automatically get ntdll's crt. IMO ntdll_crt should only be linked into native dlls/exes, baed on the module type.